pkg/controller/storagecluster/reconcile.go

```diff
@@ -294,11 +299,30 @@ func (r *ReconcileStorageCluster) reconcileNodeTopologyMap(sc *ocsv1.StorageClus
 						updated = true
 					}
 				}
+				if strings.Contains(label, "rack") {
```

Comment:
should this label match be a little more unique than  just "rack". It seems possible there could be an unintended collision here with an unrelated node label.

Reply:
I want to leave it open for now. If it becomes an issue we can restrict it later.

Reply:
this is going to match anything with the substring "rack" in it. those labels are arbitrary.  could we at least make it something like `ocs.openshift.io/rack`?